stream_alert/rule_processor/handler.py

```diff
@@ -54,8 +61,8 @@ def __init__(self, context, enable_alert_processor=True):
 
         self.metrics = Metrics('RuleProcessor', self.env['lambda_region'])
         self.enable_alert_processor = enable_alert_processor
-        self._failed_record_count = 0
-        self._alerts = []
+        self._failed_record_count = Value('I', 0)
```

Comment:
`_failed_record_count` is a slightly confusing name, pls add a comment or make it more descriptive